experiments/train_celeba_classifier.py

```diff
@@ -144,11 +144,11 @@ def run():
     monitored_quantities = [cost, accuracy]
     valid_monitoring = DataStreamMonitoring(
         monitored_quantities, valid_monitor_stream, prefix="valid",
-        before_first_epoch=False, after_epoch=False, every_n_epochs=5)
+        before_first_epoch=False, after_epoch=False, every_n_epochs=1)
```

Comment:
A simpler, equivalent setup would be `before_first_epoch=False, after_epoch=True`.